_posts/2023-04-05-ml-deployment-status.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "See machine learning model status in OpenSearch Dashboards"
+authors:
+- naarcha
+date: 2023-04-05
+categories:
+ - technical-post
+meta_keywords: machine learning, ml commons, opensearch dashboards, model-serving framework, deployed models dashboard
+meta_description: Learn how to check your deployed machine learning models deployment status inside OpenSearch Dashboards.
```

Comment:
Please update the meta with the following changes: 
meta_keywords:  machine learning, ML Commons for OpenSearch, deployed models dashboard 
meta_description: Test the experimental deployed model dashboard functionality of OpenSearch Dashboards to view the responsiveness of ML nodes in your cluster.